src/random.cpp

```diff
@@ -207,6 +207,30 @@ static std::mutex cs_rng_state;
 static unsigned char rng_state[32] = {0};
 static uint64_t rng_counter = 0;
 
+void RandAddSeedSleep()
+{
+    int64_t nPerfCounter1 = GetPerformanceCounter();
+    std::this_thread::sleep_for(std::chrono::milliseconds(1));
+    int64_t nPerfCounter2 = GetPerformanceCounter();
+
+    unsigned char buf[64];
+    CSHA512 hasher;
+    // Combine with and update state
+    {
+        std::unique_lock<std::mutex> lock(cs_rng_state);
```

Comment:
Would be nice to factor out the 'add this data to the random state' to a function, to make sure the same algorithm is used everywhere.